action.yaml

```diff
@@ -27,6 +27,10 @@ inputs:
     required: false
     default: 'net5.0'
     description: 'The dotnet target framework used for the plugin build (default: net5.0)'
+  verbosity:
+    required: false
+    default: 'info'
```

Comment:
Do we really not want to default to debug?

Reply:
hmmm, fair point, and tbh there isn't much being logged in debug anyway :thinking:
But at the same time IMHO it is better to log the least by default, but offer the option to verbose logging if need be.

In the end I don't feel too strongly either way, if you want to go for more verbosity so be it ;)

Reply:
I mean, you don't care about the output until something goes wrong, and then you need to figure out *why* it went wrong.

Nice not having to mess with CI settings to get more info – that's my take on it anyways.

Reply:
```suggestion
    default: 'debug'
```